### ISSUE TITLE
Could not find method packageName() for arguments

### ISSUE THREAD

**Issue by flolom** [sjcqs/gradle-config#9]
Reproduced in the test/sample projects of this repository, during the sync of the project : 

```
Could not find method packageName() for arguments [task ':test_lib:generateDebugBuildConfig' property 'buildConfigPackageName'] on task ':test_lib:generateDebugSettings' of type com.sjcqs.android.config.task.GenerateSettingsTask.
```

This occurs when upgrading the gradle wrapper version to version `6.1.1` and the android gradle plugin to `4.0.0`

**Comment by sjcqs** [comment]
Android Gradle Plugin breaking change I guess. 🤷

**Comment by sjcqs** [comment]
Should be fixed by version 1.0.0
